tests/model_registry/model_catalog/upgrade/test_catalog_upgrade.py

```diff
@@ -0,0 +1,301 @@
+import pytest
```

Comment:
Can we move this to tests/model_registry/upgrade please? MC is currently part of MR, so the upgrade tests can be together.

Reply:
I prefer keeping registry and catalog separate as it can be useful for e.g. parallel runs like we said yesterday

Reply:
Parallel execution won't have any impact on moving it. Upgrade tests should be kept together. We don't need two different runs of upgrades. MR and MC can be tested together.